common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.hal.testsuite.page.config.elytron;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.shared.util.ResourceManager;
+import org.jboss.hal.testsuite.page.ConfigPage;
+import org.jboss.hal.testsuite.util.PropUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public abstract class AbstractElytronConfigPage<PARTICULAR_PAGE extends AbstractElytronConfigPage<PARTICULAR_PAGE>>
+        extends ConfigPage {
+
+    public abstract PARTICULAR_PAGE navigateToApplication();
+
+    public boolean resourceIsPresentInMainTable(String resourceName) {
+        return getConfigFragment().resourceIsPresent(resourceName);
+    }
+
+    public ConfigFragment switchToConfigAreaTab(final String tabName) {
+        return getConfig().switchTo(tabName);
+    }
+
+    public boolean resourceIsPresentInConfigAreaTable(String resourceName) {
+        return resourceIsPresentInConfigAreaTable(resourceName, 0);
+    }
+
+    public boolean resourceIsPresentInConfigAreaTable2ndColumn(String resourceName) {
+        return resourceIsPresentInConfigAreaTable(resourceName, 1);
+    }
+
+    private boolean resourceIsPresentInConfigAreaTable(String resourceName, int tableColumnIndex) {
+        By configAreaSelector = ByJQuery.selector("." + PropUtils.get("configarea.class") + ":visible");
+        WebElement configAreaRoot = getContentRoot().findElement(configAreaSelector);
+        ConfigFragment configFragment = Graphene.createPageFragment(ConfigFragment.class, configAreaRoot);
```

Comment:
If this part is general, it might be worth extracting it to some general page which contains config area, just a suggestion.